api/src/main/java/org/apache/cloudstack/api/command/admin/usage/ListUsageRecordsCmd.java

```diff
@@ -81,6 +82,10 @@
     @Parameter(name = ApiConstants.INCLUDE_TAGS, type = CommandType.BOOLEAN, description = "Flag to enable display of Tags for a resource")
     private Boolean includeTags;
 
+    @Parameter(name = ApiConstants.OLD_FORMAT, type = CommandType.BOOLEAN, description = "Flag to enable description rendered in old format with no processing")
```

Comment:
should this have a default (of true)?

Reply:
1. old behaviour should be the default, unless the documentation is (somewhere...) updated.
2. word "processing" - does this mean that there will be additional CPU usage and DB hammering on top of existing (I know some people's usage_jobs taking many hours to complete - would like to understand performance consequences)

Reply:
I'll update the docs. Will fix description.